Drivers/Src/Switches.c

```diff
@@ -1,7 +1,12 @@
 #include "Switches.h"
+#include "stm32f4xx.h"
+#include "os.h"
+#include "Tasks.h"
```

Comment:
I'm not a huge fan of having the driver layer depend on the apps layer.

I think it's beyond the scope of this pull request to fix this though, since this is relatively common in the codebase.